### PR TITLE
Defect check/issue1206 check

### PR DESCRIPTION
**Pull request by LuxoftAKutsan**
Check Issue https://github.com/SmartDeviceLink/sdl_core/issues/1206

### CODE REVIEW

File: test_scripts/Defects/4_5/1206_REQUEST_PTU_Trigger_PTU_failed_previous_IGN_ON.lua [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---------------------------------------------------------------------------------------------
+-- Script verifies issue https://github.com/SmartDeviceLink/sdl_core/issues/1206
+-- Check that in state UPDATE_NEEDED after ignition cycle SDL will try to perform PolicyUpdate
+---------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require("user_modules/script_runner")
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+local mobile_session = require("mobile_session")
+local atf_logger = require("atf_logger")
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_5/1206_REQUEST_PTU_Trigger_PTU_failed_previous_IGN_ON.lua]: remove. is not used.

File: test_scripts/Defects/4_5/1206_REQUEST_PTU_Trigger_PTU_failed_previous_IGN_ON.lua [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+            EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered",
+                                   {application = {appName = config.application1.registerAppInterfaceParams.appName}})
+            EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"}):Do(function()
+              commonFunctions:userPrint(Color.blue, "Received OnStatusUpdate:UPDATE_NEEDED")
+            end)
+        end)
+end
+
+function ignition_off(self)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_5/1206_REQUEST_PTU_Trigger_PTU_failed_previous_IGN_ON.lua]: better to use local function.

**Comment by GetmanetsIrina** [comment]
Remove not actual 1880_DefaultAndRPCTimeoutGENERIC_ERROR.lua from PR.

**Comment by dboltovskyi** [comment]
This PR is closed.
Script for 1206 moved to https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1762
Script for 1880 moved to https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1766

**Pull request closed without merging**